src/epd5in83b_v2/graphics.rs

```diff
@@ -0,0 +1,189 @@
+use crate::color::TriColor;
+use crate::epd5in83b_v2::{DEFAULT_BACKGROUND_COLOR, HEIGHT, NUM_DISPLAY_BITS, WIDTH};
+use crate::graphics::{DisplayColorRendering, DisplayRotation};
+use crate::prelude::TriDisplay;
+use embedded_graphics_core::prelude::*;
+
+/// Full size buffer for use with the 5in83 EPD
+///
+/// Can also be manually constructed:
+/// `buffer: [DEFAULT_BACKGROUND_COLOR.get_byte_value(); 2 * NUM_DISPLAY_BITS as usize]`
+pub struct Display5in83 {
+    buffer: [u8; 2 * NUM_DISPLAY_BITS as usize],
+    rotation: DisplayRotation,
+}
+
+impl Default for Display5in83 {
+    fn default() -> Self {
+        let mut display = Display5in83 {
+            buffer: [DEFAULT_BACKGROUND_COLOR.get_byte_value(); 2 * NUM_DISPLAY_BITS as usize],
+            rotation: DisplayRotation::default(),
+        };
+        // We need to invert chromatic part to black so it will be render white
+        let offset = display.chromatic_offset();
+        display.buffer[offset..].fill(0x00);
+        display
+    }
+}
+
+impl DrawTarget for Display5in83 {
+    type Color = TriColor;
+    type Error = core::convert::Infallible;
+
+    fn draw_iter<I>(&mut self, pixels: I) -> Result<(), Self::Error>
+    where
+        I: IntoIterator<Item = Pixel<Self::Color>>,
+    {
+        for pixel in pixels {
+            self.draw_helper_tri(WIDTH, HEIGHT, pixel, DisplayColorRendering::Negative)?;
+        }
+        Ok(())
+    }
+}
+
+impl OriginDimensions for Display5in83 {
+    fn size(&self) -> Size {
+        Size::new(WIDTH, HEIGHT)
+    }
+}
+
+impl TriDisplay for Display5in83 {
+    fn buffer(&self) -> &[u8] {
+        &self.buffer
+    }
+
+    fn get_mut_buffer(&mut self) -> &mut [u8] {
+        &mut self.buffer
+    }
+
+    fn set_rotation(&mut self, rotation: DisplayRotation) {
+        self.rotation = rotation;
+    }
+
+    fn rotation(&self) -> DisplayRotation {
+        self.rotation
+    }
+
+    fn chromatic_offset(&self) -> usize {
+        NUM_DISPLAY_BITS as usize
+    }
+
+    fn bw_buffer(&self) -> &[u8] {
+        &self.buffer[0..self.chromatic_offset()]
+    }
+
+    fn chromatic_buffer(&self) -> &[u8] {
+        &self.buffer[self.chromatic_offset()..]
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::color::TriColor::Black;
+    use crate::color::TriColor::Color;
```

Comment:
I assume this should be Chromatic or White?

Reply:
It shouldn' be there at all. btw, i'm not able to compile library/tests on the host cuase it's tied to unix system, while i'm on windows

Reply:
What is the error message when compiling the library under windows? Besides the examples which are targeted for an easy start embedded linux (e.g raspberry pis), everything else should compile nearly everywhere.

Could you maybe even make a seperate issue for this?

Reply:
#94  and i will try to find some time after christmas to fix it